tutorial/README.md

```diff
@@ -341,6 +342,28 @@ db2 "INSERT INTO DB2INST1.CUSTOMERS(first_name, last_name, email) VALUES ('John'
 docker-compose -f docker-compose-db2.yaml down
 ```
 
+## Using Cassandra
+
+```shell 
+# Start the topology as defined in https://debezium.io/docs/tutorial/
+export DEBEZIUM_VERSION=1.3
+
+docker-compose -f docker-compose-cassandra.yaml up
+
+# Consume messages from a Debezium topic
+docker-compose -f docker-compose-cassandra.yaml exec kafka /kafka/bin/kafka-console-consumer.sh \
+    --bootstrap-server kafka:9092 \
+    --from-beginning \
+    --property print.key=true \
+    --topic server1.dbo.customers
+
+# Initialize and insert some test data 
+docker-compose -f docker-compose-cassandra.yaml exec cassandra bash -c 'cqlsh --file="/debezium/inventory.cql"'
+
+# Shut down the cluster
```

Comment:
Please see https://github.com/debezium/debezium-examples/tree/master/tutorial#using-postgres and add an equivalent of step `Modify records in the database via Postgres client`

Reply:
This still seems to be missing? I.e. the actual CQL statements within cqlsh for inserting/updating some records, akin to what we do for PG and the others?

Reply:
Should I add some CQL statements (Insert, Update, etc) in the README?  
I have already added the step for ` Modify records in the database via Cassandra client` where the user can do any modifications of their choice within the cqlsh under keyspace `testdb`.

Reply:
@gunnarmorling We provide it only few of dbs like MongoDB. Definitely not in postgres I pointed to @ani-sha like an example.

Reply:
> Should I add some CQL statements (Insert, Update, etc) in the README?

Yes, exactly. I thought we'd consistently do that for the others, but turns out we don't :)

Reply:
I've pushed a commit for adding some statements (see above).